cluster/charts/crossplane/crds/apiextensions.crossplane.io_compositeresourcedefinitions.yaml

```diff
@@ -108,7 +108,7 @@ spec:
                           description: type is an OpenAPI type definition for this column. See https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types for details.
                           type: string
                       required:
-                      - JSONPath
+                      - jsonPath
```

Comment:
Note that this is the only reason this PR is labelled as a breaking change - `additionalPrinterColumns.JSONPath` changed to `additionalPrinterColumns.jsonPath` with v1 CRDs. If we want to avoid the breaking change we could define our own `AdditionalPrinterColumns` struct with the old struct tag that doesn't comply with API conventions.

I'm inclined to make this break because I'd like to make sure we comply with API conventions when we hit v1beta1, and because I suspect relatively few folks are using this functionality. Presumably there's an upgrade path for those who are, i.e.:

* Remove printer columns from your XRDs
* Upgrade
* Re-add printer columns to your XRDs